include/itkMorphologicalContourInterpolator.hxx

```diff
@@ -127,11 +162,255 @@ void MorphologicalContourInterpolator<TImage>::DetermineSliceOrientations()
   }
 }
 
+template<class TImage>
+typename TImage::Pointer MorphologicalContourInterpolator<TImage>::RegionedConnectedComponents
+(typename TImage::RegionType region, typename TImage::PixelType label, itk::IdentifierType &objectCount)
+{
+  typedef BinaryThresholdImageFilter<typename TImage, BoolImageType> BinarizerType;
+  BinarizerType::Pointer bin = BinarizerType::New();
```

Comment:
Create a class member for the BinaryThresholdImageFilter and the ConnectedComponentImageFilter, then create them in the constructor with ::New().  Contructing these objects with ::New() is a relatively heavy weight operatior.